internal/plugin/connectors/tcp/mssql/connection_details.go

```diff
@@ -46,12 +54,15 @@ func NewConnectionDetails(credentials map[string][]byte) *ConnectionDetails {
 	switch {
 	case bytes.Equal(sslMode, []byte("disable")):
 		connDetails.SSLMode = string(credentials["sslmode"])
+		break
 	case bytes.Equal(sslMode, []byte("enable")):
+		connDetails.parseSSLOptions(credentials)
 		fallthrough
 	case bytes.Equal(sslMode, []byte("verify-ca")):
+		connDetails.parseSSLOptions(credentials)
 		fallthrough
```

Comment:
I think you can stack two case statements together: https://www.golangprograms.com/golang-switch-case-statements.html under "Switch multiple cases Statement"